contracts/dapis/DapiServer.sol

```diff
@@ -996,6 +970,32 @@ contract DapiServer is
         }
     }
 
+    /// @notice Called privately to aggregate the Beacons and return the result
+    /// @dev Tha aggregation of Beacons may have a different value than the
+    /// respective Beacon set, e.g., because the Beacon set has been updated
+    /// using signed data
+    /// @param beaconIds Beacon IDs
+    /// @return value Aggregation value
+    /// @return timestamp Aggregation timestamp
+    function aggregateBeacons(
+        bytes32[] memory beaconIds
+    ) private view returns (int224 value, uint32 timestamp) {
```

Comment:
I remember a discussion about this contract being sort of a base contract for other use cases of DapiServer in the future. Do you think this function might be useful in the contracts inheriting from this one? If so, should we make this function internal instead?

Reply:
It feels like it's more likely that we will either not modify DapiServer or the modifications needed will be significant enough that we will not be able to reuse the current contracts without modification, and the things we do to make them more reusable will do nothing but reduce readability. I will see if these can be addressed in a final PR.